packages/funding-rate-proposer/index.js

```diff
@@ -20,7 +20,6 @@ const { getWeb3 } = require("@uma/common");
  * @notice Runs strategies that request and propose new funding rates for Perpetual contracts.
  * @param {Object} logger Module responsible for sending logs.
  * @param {Object} web3 web3.js instance with unlocked wallets used for all on-chain connections.
- * @param {String} perpetualAddress Contract address of the Perpetual Contract.
```

Comment:
Unused variable in this bot, perp addresses auto detected by querying PerpFactory past events